admin/src/main/scala/io/buoyant/admin/names/DelegateApiHandler.scala

```diff
@@ -158,29 +159,29 @@ object DelegateApiHandler {
     def writeBuf[T](t: T): Buf = Buf.ByteArray.Owned(mapper.writeValueAsBytes(t))
   }
 
+  def getDelegateRsp(dtab: String, path: String, interpreter: NameInterpreter): Future[Response] =
```

Comment:
agree with you both, ticketed here:
https://github.com/BuoyantIO/linkerd/issues/326

i thought about going down this route, but decided to limit the scope of this PR to just not introducing any _more_ dependencies.